### DISCUSSION TITLE
The gradle command failed.

### DISCUSSION THREAD

**Discussion by jayahandikadarussalam** [skiptools/skip#169] [General]
Hello, i just downloaded skip showcase project, when i tried to run it always give me an error like : 
ERROR: JAVA_HOME is set to an invalid directory: /opt/homebrew/opt/java

Please set the JAVA_HOME variable in your environment to match the
location of your Java installation.

The gradle command failed. Review the log for details and consult https://skip.tools/docs/faq for common solutions. Command: gradle -p ../Android launchDebug

**Comment by marcprux** [comment]
Does the `/opt/homebrew/opt/java` folder exist? That should be where Java was installed as part of the Homebrew installation.

What does the output of `skip checkup` look like?

**Reply by jayahandikadarussalam** [reply]
Hello sir, the following are the results of skip checkup :
<img width="1014" alt="image" src="https://github.com/skiptools/skip/assets/25688986/d67ded05-34fc-406e-8ae1-a0ea12d28d48">

**Reply by marcprux** [reply]
It looks like the issue may be related to the output:

> [!] Java version 11.0.17 (< 21.0.0) (check JAVA_HOME environment: /Library/Java/JavaVirtualMachines/jdk-11.0.17.jdk/Contents/Home)

Do you have the JAVA_HOME environment variable set in your ~/.zshrc or ~/.bashrc file? If so, can you remove it and try `skip checkup` again in a new Terminal window?

If it still fails, can you attach the output log mentioned in the `skip checkup` command, as well as the result of the command `env` and `which java`?

**Reply by jayahandikadarussalam** [reply]
Yes i have JAVA_HOME env for Intellij Idea in .zshrc : 
<img width="1136" alt="image" src="https://github.com/skiptools/skip/assets/25688986/f654d678-6800-40c9-a07c-b328faa0d4a6">
<img width="1006" alt="image" src="https://github.com/skiptools/skip/assets/25688986/b98f02c8-80fc-47d9-b560-602a5b69cfb5">
<img width="1009" alt="image" src="https://github.com/skiptools/skip/assets/25688986/0b0fa761-382f-42b1-b685-a8e3fa0622ba">
<img width="1136" alt="image" src="https://github.com/skiptools/skip/assets/25688986/9002bea2-de1d-42de-a373-9e001034ff50">

**Reply by marcprux** [reply]
Skip requires JDK 21 or higher, and your JAVA_HOME is overriding the auto-detection of the Homebrew installation's JDK that we prefer. Can you remove JAVA_HOME from your environment and try `skip checkup` again in a new terminal?
